pubspec.yaml

```diff
@@ -1,5 +1,6 @@
 name: sync_http
-version: 0.1.4
+version: 0.2.0
+publish_to: none
```

Comment:
What's this for?

Reply:
Initially, this is added to indicate this not ready for publish.
https://github.com/dart-lang/http_multi_server/commit/98c2789fc1c47544afc8efbec5fa9c1499ecf5e2

But I think it is ok to remove it.